### PR TITLE
Add functionality to add basic lands

### PR DESCRIPTION
**Pull request by Frost36**

### CODE REVIEW

File: src/landAdder.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// This file contains components and logic related to importing and exporting from text formats.
+
+import React from 'react';
+
+function LandAdder(props) {
+  const selectorId = props.id + '-selctor';
```
Comment (uponthesun) [inline src/landAdder.js]: typo

**Review by uponthesun — COMMENTED** [review]
Nice refactor. Everything looks good, the only thing is I think we should start caching the scryfall card data in-memory since this'll routinely be adding a lot of duplicate cards.

File: src/landAdder.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+// This file contains components and logic related to importing and exporting from text formats.
```
Comment (uponthesun) [inline src/landAdder.js]: Update

**Comment by Frost36** [comment]
Agreed on the caching, that should be easy to add. We can do that in a separate CR.

**Review by uponthesun — APPROVED** [review]

**Pull request merged**